deploy/deploy.ts

```diff
@@ -40,7 +40,7 @@ export default async function (): Promise<void> {
     implementation = await deployContract(
         hre,
         'ClaveImplementation',
-        [await batchCaller.getAddress()],
+        [await eoaValidator.getAddress()],
```

Comment:
Should the line above still be calling "ClaveImplementation"?

Reply:
no - looks like I forgot to commit changes to the deploy script. Will get this cleaned up later